website/src/pages/Status/Status.jsx

```diff
@@ -36,6 +37,12 @@ const Status = () => {
     isLoading: isLoadingPreviousExe,
     error: errorPreviousExe,
   } = useApiCall(`${import.meta.env.VITE_API_URL}recent`);
+  const {
+    data: dataStatusStats,
+    isLoading: isLoadingStatusStats,
+    error: errorStatusStats,
+  } = useApiCall(`${import.meta.env.VITE_API_URL}status/stats`);
+  console.log(dataStatusStats);
```

Comment:
I think we can get rid of this `console.log` i am assuming it was added for debugging purposes